content/en/Getting started/glossary.md

```diff
@@ -173,11 +173,13 @@ A _Pentester_ is a Cobalt pentester who works with a _Pentest Lead_ to test a sp
 _Cobalt Staff_ members may help you manage the users in your organization. They may also help
 manage work on your pentests.
 
-## Dynamic Page
+## Compliance Audit
 
-Web applications typically include _static_ and _dynamic_ web pages. A Dynamic Page includes content
-that can be customized, either through an application server (server-side) or through code such as
-JavaScript running in the browser (client-side).
+As [defined](https://csrc.nist.gov/glossary/term/compliance_audit) by NIST, a comprehensive review of an organization's adherence to governing documents such as whether a Certification Practice Statement satisfies the requirements of a Certificate Policy and whether an organization adheres to its Certification Practice Statement.
+
+## Comprehensive Pentest
+
+A pentest performed for [security audit](#security-audit), [compliance audit](#compliance-audit), or customer attestation that includes comprehensive [reports](/platform-deep-dive/pentests/reports/) intended for external stakeholders.<!-- Learn more about [Comprehensive Pentests](/platform-deep-dive/pentests/pentest-types/). -->
```

Comment:
@yogipetkar I added a glossary entry about Comprehensive Pentests. IMO, since a Full Report is not the only report type that we offer for Comprehensive Pentests, should we use a broader term "comprehensive reports?"

Reply:
cc @TheInstaGrahame , what term will you be using for these pentests and reports?